roles/httpd/tasks/main.yml

```diff
@@ -146,17 +146,11 @@
   tags:
     - base
 
-- include: home-page.yml
-
 - name: place the script to generate home pages
   template: src=refresh-wiki-docs.sh
             dest=/usr/bin/iiab-refresh-wiki-docs
             mode=0755
 
-- name: generate the offline documents
```

Comment:
Minor quibble but is separating both [refresh-wiki-docs](https://github.com/iiab/iiab/blob/master/roles/httpd/templates/refresh-wiki-docs.sh) actions into 2 diff playbooks really worth it?

Makes it quite a bit harder for new contributors to understand, spread across these 2 playbooks:
* roles/httpd/tasks/main.yml
* roles/4-server-options/tasks/main.yml

Perhaps to pick 1 playbook or the other?